tests/test_commands.py

```diff
@@ -234,3 +235,9 @@ def test_docstring_injector(mapdl, method):
             assert "``str.to_list()``" in docstring
             assert "``str.to_array()``" in docstring
             assert "``str.to_dataframe()``" in docstring
+
+
+def test_string_with_literal():
+    output = StringWithLiteralRepr("asdf\nasdf")
+    assert output.__repr__()
```

Comment:
you can do this instead
`assert output.__repr__() == output`

if it were a regular string, it would not be equal

Reply:
Surprisingly it does work in the previous way.

Reply:
Done.